### PR TITLE
Document `activity_ids` and `expense_ids`

### PR DESCRIPTION
**Pull request by pic**

### CODE REVIEW

File: sections/invoices.md [hunk 1]
```diff
@@ -323,3 +323,17 @@ Mandatory fields are marked with a star (\*):
 - **tags** – ["Hosting", "Europe"]
 
 Omitted fields `service_period_from` and `service_period_to` indicates the absences of a service period.
+
+It's possible to mark as billed activities or project expenses using respectively `activity_ids` and `expense_ids` on positions
```
Comment (sunsations) [inline sections/invoices.md:327]: It's possible to mark billable activities or project expenses using respectively `activity_ids` and `expense_ids` on positions

Comment (pic) [reply sections/invoices.md:327]: @sunsations I'm not sure. They are already "billable", with this operation the are marked "as billed"

Comment (nethad) [reply sections/invoices.md:327]: Would make it active: Select activities or expenses to be billed by this invoice. IDs are selected on positions and after that, they are marked as billed.

**Comment by dennisoderwald** [comment]
We've already tested it, it works. Thank you very much for the fast implementation!

**Pull request merged**